packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.test.tsx

```diff
@@ -521,7 +521,8 @@ describe('SpinButton', () => {
     const exampleDefaultValue = '12';
 
     function delay(millisecond: number): Promise<string> {
-      return new Promise<string>(resolve => setTimeout(resolve, millisecond));
+      // tslint:disable-next-line:no-any
+      return new Promise<string>((resolve: any) => setTimeout(resolve, millisecond));
```

Comment:
Any suggestions here? I'm not sure if resolves have a type

Reply:
if possible special case the tslint validation to exclude .test and .example.